src/beacon-api/app/api/v1/cohort.py

```
@@ -23,112 +41,226 @@ async def get_cohorts(
+@router.get("/synced", response_model=SyncedCohortResponse)
+def get_synced_cohorts(
+    skip: int = 0,
+    limit: int = 100,
+    search: Optional[str] = None,
+    cohort_ids: Optional[str] = None,
+    tags: Optional[str] = None,
+    includePerformance: bool = False,
+    startDateTime: Optional[str] = None,
+    endDateTime: Optional[str] = None,
+    frequency: Optional[str] = "hourly",
+    db: Session = Depends(get_db),
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)